client/blocks/comment-detail/comment-detail-author.jsx

```diff
@@ -26,18 +37,18 @@ export class CommentDetailAuthor extends Component {
 		authorEmail: PropTypes.oneOfType( [ PropTypes.bool, PropTypes.string ] ),
 		authorId: PropTypes.number,
```

Comment:
Are we sure we're passing this through? I don't believe this is wired up in CommentDetailComment.

Since CommentDetailAuthor is now connected, maybe consider allowing it to pull in it's own data deps, or passing through click handlers. So we can keep this as a block or a simple UI component.

Reply:
Oh shoot, I guess it was a leftover from the past, and I just assumed we used it somewhere.

Reply:
Note that a larger refactor doesn't need to be done in this PR, I'm just noting that it can be cleaned up later.